### PR TITLE
Update manage-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a new link for Building Office add-ins as the old one was broken

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3641

File: microsoft-365/admin/manage/manage-deployment-of-add-ins.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Updates for add-ins happen as follows:
 
 [Manage add-ins in the admin center](manage-addins-in-the-admin-center.md)
 
-[Building Office Add-ins](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-ins-fundamentals).
+[Building Office Add-ins](https://docs.microsoft.com/office/dev/add-ins/quickstarts/word-quickstart?tabs=yeomangenerator.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
[Build your first Word task pane add-in](https://docs.microsoft.com/office/dev/add-ins/quickstarts/word-quickstart?tabs=yeomangenerator).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**